### ISSUE TITLE
APA WG recharter 2021

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/strategy#256]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://raw.githack.com/w3c/apa/charter-2021/charter.html)

This is an existing WG recharter 

Communities suggested for outreach: 

Known or potential areas of concern: This charter simplifies but does not substantially change the scope from the previous charter. It includes Rec-track deliverables for Personalization and Pronunciation technologies, including the possibility of not-yet-determined modules for those technologies. It includes a substantial set of non-normative deliverables.

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

* Email to [group-apa-chairs](mailto:group-apa-chairs@w3.org)
* [APA Issues](https://github.com/w3c/apa/issues)

Anything else we should think about as we review?

**Comment by samuelweiler** [comment]
[Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2018%2F08%2Fapa-charter&doc2=https%3A%2F%2Fraw.githack.com%2Fw3c%2Fapa%2Fcharter-2021%2Fcharter.html)

**Comment by samuelweiler** [comment]
Nice charter.  I see that the last one was 3 years, and I support this one being long, also.

I see that it [doesn't use the most recent template](https://github.com/w3c/apa/issues/130 ) as evidenced by the line re: security and privacy sections.  I'm not sure what else might be missing.  Up to you whether to backport the change(s) or start from the template.  Other than that minor cleanup, no security or privacy concerns.  

Also need to update the [history table](https://github.com/w3c/apa/issues/132). 

I'm not sure this charter lists what is intended by "Adopted Working Drafts", but I'm [confused by that item](https://github.com/w3c/w3process/issues/500), generally.

**Comment by himorin** [comment]
Queuing in i18n group for review comment, but one quick point, I think this draft need to be updated on patent policy, since PP2020 is still mentioned as pre-approval.

**Comment by himorin** [comment]
No comment/request from i18n. marking as completed.

**Comment by michael-n-cooper** [comment]
[@brewerj feedback](https://www.w3.org/mid/cec4bf4d-6b2a-dc1c-4031-126773f16ceb@w3.org)

**Comment by samuelweiler** [comment]
Sec and priv text appears to have been updated to match the template.  marking review as complete.
